app/Console/Commands/Environment/AppSettingsCommand.php

```diff
@@ -38,6 +37,7 @@ class AppSettingsCommand extends Command
      * @var string
      */
     protected $signature = 'p:environment:setup
+                            {--author= : The email that services created on this instance should be linked to.}
```

Comment:
Its just set at the environment level and replaces the current value which is a UUID. This is mostly for sharing so that it links back to a specific person or service. In most cases it'll just be the admin's email address, they can change it in the exported files, or at the environment level if needed. I don't see a reason to allow changing it on each export.

Reply:
Oh I would just store it in the database and add a field on the service, sorry, egg editor